pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -412,6 +419,10 @@ func ReadConfig(config io.Reader) (Config, error) {
 		cfg.Metadata.SearchOrder = fmt.Sprintf("%s,%s", metadata.ConfigDriveID, metadata.MetadataID)
 	}
 
+	if !supportedLBProvider[cfg.LoadBalancer.LBProvider] {
```

Comment:
And use:

```golang
if util.Contains(supportedLBProvider, cfg.LoadBalancer.LBProvider) {
    ...
}
```

Reply:
Hey, tbh I wasn't aware we had util.Contains. I have changed this now. I'll amend my commit and push it